It's music to my ears! (Literally)

Tron-117 opened the pull request:
Gets the jukebox working again, as well as adding a few tracks for selection

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

<!-- NOTE: This format is NOT REQUIRED for things like runtime fixes, code fixes and optimizations. In those instances you should try to give a description of what is being fixed or optimized but are not required to fill out the complete changelog. -->
<!-- Adjusting things like armor or weapon values for balance, while they may be 'fixes' in their own way, are not considered code fixes as described above and require the use of the Pull Request format below.-->


## About The Pull Request
Ever since DR rebased in 2021, the jukebox has been broken. Now, it works. Jukeboxes work differently since they were last properly functioning:
- The music volume adjusts based on your distance from the jukebox
- The track plays constantly instead of just stopping and restarting if you were to leave and re-enter the vicinity of the jukebox
- You can adjust the volume of the music on the jukebox itself

A new addition to the system is music tapes. These tapes will be loaded with songs which will be added to the tracklist of the jukebox when inserted, and can be taken out at any time. Ideally, the music on the tapes will be sorted by the different genres of music from the time period.

For future reference, to add more songs to the jukebox, go to sound/f13music/jukebox_music/ and add in the .ogg file of the song you want added. You have to name the songs a specific way though, or else the game won't be able to load in the tracks.
An example:
[song name]+[song category]+[length in deciseconds]+[beat in deciseconds]+[track ID].ogg
The Wanderer+Base2+1790+5+11
(A decisecond is just a second multiplied by 10. The beat of the song doesn't seem to matter, so just keep it at 5)

To note, you **MUST** put + signs in-between the different sections of the name so the game can properly load the information of the track. Putting spaces in the song name is fine.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Who doesn't love music?
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->


## Changelog
<!-- This is mostly optional for small Pull Requests, but if you value being credited for your work in-game be sure to fill it out. To opt-out, remove everything below including the start and end :cl: brackets. -->

<!-- If your Pull Request includes a minor single line variable edit, probably do not fill out this changelog. -->
<!-- However, if your pull request includes massive or immediately noticeable changes, briefly describe those changes in some way in this changelog. -->

:cl:
fix: Makes the jukebox work again
config: Re-allows for for internet sounds to be played with regular youtube links
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

thegrovellingpeasant commented:
When our Lord entered the temple and found it polluted by money-changers and beasts (which is what you are trying to add in this PR), did he ask them to leave? Did he cry? Did he simply walk away?

No. He drove them out. It is one thing to forgive a slap across my cheek, but an insult to the Lord requires... no, it demands correction. With that being said, no catgirl mormon missionaries.

Tron-117 committed:
It's music to my ears! (Literally)

Gets the jukebox working again, as well as adding a few tracks for selection

Tron-117 committed:
Adds another track to the list

Tron-117 committed:
Even more tracks

Tron-117 committed:
Greatly decreases the cooldown for the jukebox

Tron-117 committed:
Music Tapes

Adds music tapes which can be inserted into the jukebox. Will be expanded upon in the future.

Tron-117 committed:
Re-enables playing internet sounds

Works only on a live server it seems, but this should allow staff to put in a regular youtube link and play a sound.
